website/docs/developer/deployment-guide/deploying-from-windows.mdx

```diff
@@ -0,0 +1,185 @@
+---
+id: deploying-from-windows
+title: Deploy from Windows
+description: A concrete set of instructions on deploying from Windows to create a test environment.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Deploying Cumulus, CumulusDashboard, and Orca from Windows brings some additional challenges.
+The goal of this page is to provide a set of modified instructions to get around common errors.
+
+### Notes
+- Choose a PREFIX that will identify your installation when in AWS. This string will be used throughout deployment.
+- Connect to the NASA VPN to be able to connect to AWS.
+  :::warning
+  The VPN drastically slows down Terraform operations, and limits what documentation can be viewed. Switch off when applicable.
+  :::
+- Commands here will use ```us-west-2``` for region because of the current state of our sandbox. Replace consistently as needed.
+    - Make sure any operations in AWS are done under the correct region.
+
+## Application
+This application will be used in future steps to authenticate users.
+- Go to https://uat.urs.earthdata.nasa.gov/profile
```

Comment:
looks like I don't have account on this website yet. Should this be UAT?

Reply:
I'm not sure I understand the question.

Reply:
just making sure ```https://uat.urs.earthdata.nasa.gov/profile``` is the right URL.

Reply:
Yes, @rizbi-hassan you may need to sign up for an account for the UAT environment of URS.